### ISSUE TITLE
Data Being Generated in 01-Vanilla-Neural Networks Problem

### ISSUE THREAD

**Issue by joseaveldanes** [dlab-berkeley/Python-Deep-Learning-Legacy#23]
Slightly before challenge 2, we are running neural network models and looking at different ways of changing training and validation accuracy. Everyone is getting a different graph for the validation accuracy and we believe the issue is the fact that there is no random seed that is generated at the beginning to make sure we are grabbing the same subset of data in the training process. 
<img width="869" alt="Screenshot 2023-06-12 at 3 59 36 PM" src="https://github.com/dlab-berkeley/Python-Deep-Learning/assets/57539176/e2d4a71f-267d-415c-afdd-b2ba99094998">
<img width="869" alt="Screenshot 2023-06-12 at 3 59 36 PM" src="https://github.com/dlab-berkeley/Python-Deep-Learning/assets/57539176/071561d9-d695-4c5e-8783-f8f9447648e2">
